docs/resources/tanzu_kubernetes_cluster.md

```diff
@@ -7,9 +7,13 @@ Description: |-
 # Tanzu Kubernetes Cluster (Class-based Cluster) Resource
 
 This resource enables users to create and configure a TKG cluster for both TKGM & TKGS.
```

Comment:
"This resource enables users to create and configure a TKG cluster" is incorrect.
Statement should read, something on the lines of - This resource is to create your own custom ClusterClass resource for a Tanzu Kubernetes Grid 2.x introduced with vSphere 8.

Also as the norm, link to appropriate Tanzu docs for additional details - like: https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-C778E447-DDBB-49FC-B0B2-A8012AC56B0E.html and https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-41DE8F22-56B8-4669-AF3F-E4B4372BDB9E.html (if management cluster registration is available as well).

Reply:
fixed.